lib/conditionals.md

```diff
@@ -5,8 +5,12 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+Conditionals are used to control flow within a program, I think of it like a flow chart.  
+If there are two braches you use if and else.  
+If there are more than two branches you can use if, elsif, and else using as many eslifs that are needed.  
+The program starts at the beginning and runs until one of options is true. 
```

Comment:
You might be hinting at this with your answer, but I view this running "until" a condition is met as being a _different_ kind of control flow than `if, else` statements.

Can you be a little more specific? Maybe walk me through what happens when you run a (hypothetical) program with `if` and `else` blocks in it and what happens when you run a different (still hypothetical) program with...let's say a `while` loop.